pkg/ebpf/c/include/tracee/consts.h

```diff
@@ -19,17 +24,31 @@
 
 #define MAX_MEM_DUMP_SIZE   127
 
-// clang-format on
-
 #define MAX_STR_ARR_ELEM      38 // TODO: turn this into global variables set w/ libbpfgo
 #define MAX_ARGS_STR_ARR_ELEM 15
 #define MAX_PATH_PREF_SIZE    64
 #define MAX_PATH_COMPONENTS   20
 #define MAX_BIN_CHUNKS        110
 
+#define MAX_PERF_EVENT_NAME   ((MAX_PATH_PREF_SIZE > MAX_KSYM_NAME_SIZE) ? MAX_PATH_PREF_SIZE : MAX_KSYM_NAME_SIZE)
+
+#define REQUIRED_SYSTEM_LENGTH 9
+
 #define CAPTURE_IFACE (1 << 0)
 #define TRACE_IFACE   (1 << 1)
 
+// Send binary helper
+
+#define F_SEND_TYPE     0
+#define F_CGROUP_ID     (F_SEND_TYPE + sizeof(u8))
+#define F_META_OFF      (F_CGROUP_ID + sizeof(u64))
+#define F_SZ_OFF        (F_META_OFF + SEND_META_SIZE)
+#define F_POS_OFF       (F_SZ_OFF + sizeof(unsigned int))
+#define F_CHUNK_OFF     (F_POS_OFF + sizeof(off_t))
+#define F_CHUNK_SIZE    (MAX_PERCPU_BUFSIZE >> 1)
```

Comment:
Why ?

Reply:
Since they are only used there (and there is no other uses for them anywhere else), and also it is clearer if they are defined near the code where they are used

Reply:
This is a personal preference. We don't have well stablished coding guidelines. I'm preparing a linter for golang code with a well set of defined rules (so discussions like this won't be needed)... 

I think we should do the same for C/eBPF code woon (when we define what headers we want and what they should be responsible for).

Example: should we have function prototypes in single header or multiple headers ? How should we split headers ? One per eBPF object (when we have multiple) ? Or per subsystem (network, vm, filesystem), etc.

Reply:
Take a look at my last commit "fixing" headers indexing by clangd language server. It was all broken and, the way it is now, it is not granular. Example:

#include <common/network.h>

would automatically include

#include <common/common.h>

While the correct thing would be for headers to be included in the form:

#include <maps.h>
#include <common/common.h>
#include <common/network.h>

Instead of including a single file that include the others.

Yes, the change should be part of "creating multiple eBPF objects". And also I don't think "one" should define something in an issue and others abide "just because it was defined". There are best practices to be followed, using other open source projects as examples, most of the times.

Reply:
Nevertheless, my last commit in this PR puts us into a much better situation without too many changes, and also fixes language servers code parsing (clangd) for include directory search path. Please consider it.